### PR TITLE
Add a scratch project to the solution

### PR DESCRIPTION
**Pull request by JeremyKuhne**
This is to simplify experimenting and looking at repros.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/9039)

### CODE REVIEW

**Comment by JeremyKuhne** [comment]
Note that we could update the build scripts to ignore the existing files in the scratch project (see the comment in .gitignore). If someone wants to follow up with that...

File: src/System.Windows.Forms/tests/IntegrationTests/ScratchProject/Program.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace ScratchProject;
+
+// This project is meant for temporary testing and experimenting and should be kept as simple as possible.
+
+internal static class Program
+{
+    [STAThread]
```
Comment (dreddy-work) [inline src/System.Windows.Forms/tests/IntegrationTests/ScratchProject/Program.cs:10]: Any reason you can't use latest template that utilize code generation here?

Also, default is SystemAware.

Comment (JeremyKuhne) [reply src/System.Windows.Forms/tests/IntegrationTests/ScratchProject/Program.cs:10]: I deliberately want the non-default here so we look good for the per monitor in our experiments. The code generation stuff doesn't work without significant bending over backwards as we're directly referencing the projects and can't set `UseWindowsForms`. I feel it is better to keep the project as simple as possible to make it easier to understand.

**Review by dreddy-work — APPROVED** [review]

**Comment by RussKie** [comment]
I'm curious, why is `WinformsControlsTest` not good?

**Comment by Tanya-Solyanik** [comment]
I prefer minimal projects when debugging or reproing specific bugs, especially when tracing/memory monitoring is on.

**Pull request merged**